pkg/ddc/jindo/health_check.go

```diff
@@ -17,17 +17,20 @@ limitations under the License.
 package jindo
 
 import (
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/util/retry"
+
 	data "github.com/fluid-cloudnative/fluid/api/v1alpha1"
 	"github.com/fluid-cloudnative/fluid/pkg/common"
 	"github.com/fluid-cloudnative/fluid/pkg/ctrl"
 	fluiderrs "github.com/fluid-cloudnative/fluid/pkg/errors"
 	"github.com/fluid-cloudnative/fluid/pkg/utils/kubeclient"
-	corev1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/types"
-	"k8s.io/client-go/util/retry"
 )
 
 func (e *JindoEngine) CheckRuntimeHealthy() (err error) {
+	e.Log.V(3).Info("CheckRuntimeHealthy", "runtime name", e.runtimeInfo.GetName())
```

Comment:
V(3) -> V(1). It's because the current log system only supports production or debug level. V(1) indicates debug level.

Reply:
done